vscode/src/custom-recipes/CustomRecipesMenus.ts

```diff
@@ -194,10 +143,12 @@ export async function showRecipeTypeQuickPick(
     const userItem = {
         label: 'User',
         detail: 'User Recipes are accessible only to you across Workspaces',
```

Comment:
The casing here looks incorrect. This is neither Title Case (used in some VS Code dialogs) nor regular english casing.

It should either be 

```suggestion
        detail: 'User recipes are accessible only to you across workspaces',
```

or when it's shown in the command palette 

```suggestion
        detail: 'User Recipes Are Accessible Only to You Across Workspaces',
```

Reply:
Ah I was thinking [Workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces#_workspace-file) and _User Recipes_ are the feature / product names, and therefore are considered as [proper nouns](https://www.grammarly.com/blog/proper-nouns/) which should always be capitalized in a (non-title) sentence. Do you think those rules apply in this case? 👀

Reply:
Interesting. I'll refer to @toolmantim then, he's the native speaker 🙈